addons/portal/security/ir.model.access.csv

```diff
@@ -2,3 +2,4 @@
 "access_portal_share","access.portal.share","model_portal_share","base.group_partner_manager",1,1,1,0
 "access_portal_wizard","access.portal.wizard","model_portal_wizard","base.group_partner_manager",1,1,1,0
 "access_portal_wizard_user","access.portal.wizard.user","model_portal_wizard_user","base.group_partner_manager",1,1,1,0
+"access_portal_entry","access.portal.entry","model_portal_entry","base.group_user",1,1,1,1
```

Comment:
Beware that I don't think it is good that all connected user have full edition rights to break your portal :wink: Probably only website "designer" should have this right + another specific manager portal right (don't know about portal enough to know, to investigate).